Sources_v3/StreamChat/Workers/ChannelUpdater.swift

```diff
@@ -103,6 +104,7 @@ class ChannelUpdater<ExtraData: ExtraDataTypes>: Worker {
         command: String?,
         arguments: String?,
         attachments: [_ChatMessageAttachment<ExtraData>.Seed] = [],
+        quotedMessageId: MessageId?,
```

Comment:
... here it can stay as is since this is not public API and we can make sure we handled all edge cases.